bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/queries.ts

```diff
@@ -9,4 +9,34 @@ const LAST_SEEN_MUTATION = gql`
   }
 `;
 
+export interface BasicUserInfoSubscriptionResponse {
+  user: {
+    userId: string;
+    name: string;
+    isModerator: boolean;
+    color: string;
+    avatar: string;
+    presenter: boolean;
+  }[];
+}
+
+export const BASIC_USER_INFO = gql`
+  subscription UserNames {
+    user(
+      order_by: [
+        { presenter: desc },
+        { role: asc },
+        { nameSortable: asc },
+      ],
+    ) {
+      userId
+      name
+      isModerator
+      color
+      avatar
+      presenter
+    }
+  }
+`;
```

Comment:
It might become heavy in meetings with a lot of users.
You should consider making it as a query filtering the name with the letters that was typed and with `limit: 30`.